_posts/newsletter/2020-08-06-this-month-in-solid.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-08-06
+permalink: /newsletter/2020-08-06
+tags: [monthly, updates]
+categories: [Updates]
+---
+{% include twis-intro.md %}
+
+## Updates
```

Comment:
This one looks a bit empty? https://solid.github.io/newsletter/2020-08-06

Reply:
ah yes, deleted it and will re-do on Friday when the newsletter is ready for review